spring-integration-hazelcast/src/main/java/org/springframework/integration/hazelcast/message/HazelcastHeaders.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.integration.hazelcast.message;
+
+/**
+ * Hazelcast Message Headers
+ *
+ * @author Eren Avsarogullari
+ * @since 1.0.0
+ */
+public abstract class HazelcastHeaders {
+
+	private static final String PREFIX = "hazelcast_";
+
+	public static final String EVENT = PREFIX + "event";
+
+	public static final String MEMBER = PREFIX + "member";
```

Comment:
As well as using `Member` for this header is bad, too:
```
* Cluster member interface. The default implementation
 * {@link com.hazelcast.instance.MemberImpl} violates the Java Serialization contract.
 * It should be serialized/deserialized by Hazelcast.
```
Seems for me `getSocketAddress()` should be enough. We always can retrieve the real `Member` using 
````java
Memeber memberForMessage = HazelcastInstance.getCluster()
             .getMembers()
             .stream()
             .filter(member -> member.getSocketAddress().equal(message.getHeaders().get(HazelcastHeaders.MEMBER, InetSocketAddress.class)))
             .get();
````